php/class-fm-bylines.php

```diff
@@ -269,6 +307,28 @@ function add_byline_meta_box( $type = 'author', $label = null, $args = array() )
 		}
 
 		/**
+		 * Only set the default for the first element shown.
+		 */
+		public function display_default_byline( $classes, $name, $field ) {
```

Comment:
I think I understand why this step has to happen during this filter, but it might be worth an explanatory comment.